src/components/Steps/index.js

```diff
@@ -1,21 +1,25 @@
 import React from 'react';
 import ReactTooltip from 'react-tooltip';
+import {connect} from 'react-redux';
 
 import ScrollMouseIcon from './../ScrollMouseIcon';
 import SelectCityStep from './SelectCity';
 import SearchStep from './Search';
 import CriteriaStep from './Criteria';
+import MapStep from './Map';
 import scrollToComponent from 'react-scroll-to-component';
```

Comment:
Please move scrollToComponent up (should be first since it's library). I know it's my code, but please do that :P